### PR TITLE
Pouches i guess

### PR DESCRIPTION
**Pull request by QuakeIV**
## About The Pull Request

Adds pouches, also kills off behaviour that prevented them from behaving like backpacks.  (you can properly click them and they pop open their inventory, rather than going into your hand.  the actual pouches were easy, finding the stupid stuff i removed in storage.dm took me days and days)

## Why It's Good For The Game

Kmc told me to do it, I have no conception of why anyone would want pouches other than they are cool and store stuff.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: nsv13.dme [hunk 1]
```diff
@@ -1,2931 +1,2932 @@
-// DM Environment file for tgstation.dme.
```
Comment (QuakeIV) [inline nsv13.dme]: i have no idea why the dif had a siezure like this, its literally an identical file except for one line

File: code/game/objects/items/storage/pouch.dm [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+	w_class = WEIGHT_CLASS_BULKY //does not fit in backpack
+	slot_flags = ITEM_SLOT_POCKET
+	var/fill_type
+	var/fill_number = 0
+
+/obj/item/storage/pouch/Initialize()
+	. = ..()
+	if(fill_number && fill_type)
+		for(var/i in 1 to fill_number)
+			new fill_type(src)
```
Comment (Qustinnus) [inline code/game/objects/items/storage/pouch.dm]: does this actually work

Comment (QuakeIV) [reply code/game/objects/items/storage/pouch.dm]: not really sure, more or less just lifted it from TGMC and changed the stuff that needed to be changed to get it minimally working.

Comment (QuakeIV) [reply code/game/objects/items/storage/pouch.dm]: to clarify, the pouches work fine, not sure if this particular code actually does anything though

Comment (Qustinnus) [reply code/game/objects/items/storage/pouch.dm]: I doubt it, you're putting them inside of the pouch, theoretically making sense but we store items in the component, not the contents. (afaik)

Comment (Qustinnus) [reply code/game/objects/items/storage/pouch.dm]: also tgmc != TG so do not use them as example on most things.

File: code/datums/components/storage/storage.dm [hunk 3]
```diff
@@ -733,19 +733,6 @@
 	if(rustle_sound)
 		playsound(A, "rustle", 50, 1, -5)
 
-	if(ishuman(user))
```
Comment (Kmc2000) [inline code/datums/components/storage/storage.dm]: Are you sure about this?
Standard storage behaviour would probably be fine on pouches (clickdrag to take out of storage, click to open)

Comment (QuakeIV) [reply code/datums/components/storage/storage.dm]: its how pouches work everywhere else, and is in my experience a lot easier

Comment (QuakeIV) [reply code/datums/components/storage/storage.dm]: basically all this code does is disabling normal slot behaviour for pockets

Comment (QuakeIV) [reply code/datums/components/storage/storage.dm]: the code i removed disabled default slot behaviour for pockets, which I have removed here

File: code/game/objects/items/storage/pouch.dm (hunk 2 above)
Comment (QuakeIV) [reply code/game/objects/items/storage/pouch.dm]: i mean, i can just delete that part if you want, i didnt even really understand what purpose it served

Comment (QuakeIV) [reply code/game/objects/items/storage/pouch.dm]: i have killed that code, as well as a bit of other stuff that i didnt understand the purpose of (the only place where the fill stuff was used was the medkit pouch, which i switched to a populatecontents function).  is this better?

Comment (QuakeIV) [reply code/game/objects/items/storage/pouch.dm]: also fixed the examine function

File: code/datums/components/storage/storage.dm (hunk 3 above)
Comment (QuakeIV) [reply code/datums/components/storage/storage.dm]: switched to commenting it out per the discord conversation

**Comment by Kmc2000** [comment]
Is this ready?

**Comment by QuakeIV** [comment]
I believe so

**Comment by QuakeIV** [comment]
just caught up with master again

**Pull request merged**